### DISCUSSION TITLE
Advanced SSH and Web Terminal

### DISCUSSION THREAD

**Discussion by corneel27** [tteck/Proxmox#3820] [Q&A] [answered]
I don't have Promox for myself, but I am the maintainer of an HA-addon DAO (https://github.com/corneel27/day-ahead).
There are a few users with Promox and a problem running the addon.
They also could not run the addon "Advanced SSH & Web Terminal" (in userprofile is advanced mude enabled). 
Is it possible to install and run the advanced - addon and get access to HA-OS?

**Comment by tteck** [comment, marked as answer]
If you're running Home Assistant OS as a virtual machine in Proxmox, it is fully isolated from the Proxmox host environment. Therefore, issues occurring within Home Assistant OS are unlikely to be caused by Proxmox itself. The VM operates independently with its own system environment, and any problems are typically internal to the VM or related to its configuration.
